internal/importer/xml.go

```diff
@@ -112,24 +114,52 @@ func (x *XMLImporter) parse() error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to open %s", xmlPath)
 	}
+	// converts `@TABLEAU` comments to a document. e.g.:
+	// <!-- @TABLEAU
+	// <xxxTag>
+	// 	...
+	// </xxxTag>
+	// -->
+	metaBeginRegexp := regexp.MustCompile(`^<!--\s+@TABLEAU`)
+	metaEndRegexp := regexp.MustCompile(`-->$`)

```

Comment:
## Low performance
You can init the `regexp` in `func init(){}` only once!
## Not good`regex`
- `^<!--\s+@TABLEAU` -> `<!--\s*@TABLEAU`
- `-->$` -> `-->\s*`

## Unit test

Reply:
Fixed. Please resolve conversation.